backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,137 @@
+from random import randint
+from typing import Dict, List
+
+from String import isalpha
+
+MAX_GUESSES = 6
+WORD_LIST = [
```

Comment:
We should put this word list in a separate file

Reply:
Will do, ill save this one for last